### PR TITLE
[hotfix] custom readiness/liveness

### PR DESCRIPTION
**Pull request by rchengvariant**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes [#8](https://usxtech.atlassian.net/browse/CLOUD-8)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Sanity Testing

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: charts/variant-api/templates/deployment.yaml [hunk 1]
```diff
@@ -73,17 +73,8 @@ spec:
               containerPort: {{ .Values.service.healthCheckPort }}
               protocol: TCP
             {{- end }}
-          {{ $port := ternary "http" "health" (empty .Values.service.healthCheckPort) }}
```
Comment (genghis-tuan) [inline charts/variant-api/templates/deployment.yaml:76]: Is this port still needed?

File: charts/variant-api/values.yaml [hunk 2]
```diff
@@ -161,4 +161,18 @@ securityContext:
...
   # -- (bool) Requires that containers must run with a read-only root filesystem (i.e. no writable layer)
-  readOnlyRootFilesystem: false
\ No newline at end of file
+  readOnlyRootFilesystem: false
+
+# -- See [Probe](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#Probe) docs
+readinessProbe:
+  httpGet:
+    path: /health
+    port: 80
```
Comment (genghis-tuan) [inline charts/variant-api/values.yaml:170]: Do we also need to default `initialDelaySeconds` and `periodSeconds` for readinessProbe or is it not needed?

**Review by vibindaniel — APPROVED** [review]

File: charts/variant-api/values.yaml (hunk 2 above)
Comment (rchengvariant) [reply charts/variant-api/values.yaml:170]: good catch.

File: charts/variant-api/templates/deployment.yaml (hunk 1 above)
Comment (rchengvariant) [reply charts/variant-api/templates/deployment.yaml:76]: Not needed. Defaulted to HTTP (80)

**Pull request merged**